conans/test/functional/generators/virtualenv_python_test.py

```diff
@@ -0,0 +1,84 @@
+import platform
+import unittest
+import os
+
+from conans.util.files import load
+
+from conans.test.utils.tools import TestClient
+
+
+class VirtualEnvPythonGeneratorTest(unittest.TestCase):
```

Comment:
Should we add a test for the `None` case mentioned above?

Reply:
Sure!